src/site/static/eligibility.liquid

```diff
@@ -1502,8 +1507,25 @@ pageClass: "page-public-assistance"
   }
 
   function feraEligible() {
-    // TODO
-    return true;
+    const grossLimit = MonthlyIncomeLimit.fromAnnual([
```

Comment:
I have not always been doing this myself (I need to go back and fix it in some places) but could you add a comment with the data source used for these numbers?

Just as simple as:

`// https://www.cpuc.ca.gov/industries-and-topics/electrical-energy/electric-costs/care-fera-program`